app/polymer/src/client/preview/render.js

```diff
@@ -33,9 +33,13 @@ export default function renderMain({
     // Since we reuse `rootElement` for all stories, remove the stored instance first.
     // But forceRender means that it's the same story, so we want too keep the state in that case.
     if (!forceRender) {
-      render(html``, rootElement);
+      rootElement.innerHTML = '<div id="root-inner"></div>';
+    } else if (!rootElement.querySelector('[id="root-inner"]')) {
```

Comment:
shouldnt this be?
```
if(!forceReRender || !rootElement.querySelector('[id="root-inner"]')) {
 ...
}
```

Reply:
You're right, i made this adjustment, good to merge?